wiki/Interface.md

```diff
@@ -2884,7 +2884,7 @@ Whenever this name appears, you may substitute this with whatever name fits your
    - a function returning the initial value of the information (in this case, `initialExampleInfo`).
 
 3. **Add it to the feature-mapping**\
-   Now, in the `feature.ts` file in `src/statistics/features`, add your feature to the `ALL_FEATURES` object.
+   Now, in the `feature.ts` file in `statistics/src7features`, add your feature to the `ALL_FEATURES` object.
```

Comment:
what are these changes for? we should at least fix the `7` and in general we should make sure these happen in a separate wiki-only pr.